### PR TITLE
add new package aibs-circuit-converter

### PR DESCRIPTION
**Pull request by asanin-epfl**
Hi! Please review.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-aibs-circuit-converter/package.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from spack import *
+
+
+class PyAibsCircuitConverter(PythonPackage):
+    """Pythonic API for conversion between Allen Institute and BBP"""
+
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/nse/aibs-circuit-converter"
+    git      = "ssh://bbpcode.epfl.ch/nse/aibs-circuit-converter"
+
+    version('develop', branch='master')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-aibs-circuit-converter/package.py:15]: Can we add a version/tag here, please?

File: var/spack/repos/builtin/packages/py-aibs-circuit-converter/package.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-aibs-circuit-converter/package.py]:
```suggestion
# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

**Review by matz-e — CHANGES_REQUESTED** [review]

**Comment by asanin-epfl** [comment]
@matz-e Should I update License date in https://github.com/BlueBrain/spack/blob/develop/var/spack/repos/builtin/packages/py-neuron-reduce/package.py#L1 ? I've merged it already unfortunately.

File: var/spack/repos/builtin/packages/py-aibs-circuit-converter/package.py (hunk 1 above)
Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/py-aibs-circuit-converter/package.py:15]: done

File: var/spack/repos/builtin/packages/py-aibs-circuit-converter/package.py (hunk 2 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-aibs-circuit-converter/package.py]: I don't see the change?

**Review by matz-e — APPROVED** [review]

**Pull request merged**